extensions/ViewController.swift

```diff
@@ -12,7 +12,10 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+
         // Do any additional setup after loading the view, typically from a nib.
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tdrk18/swift-extensions/blob/22739ce465911863e00c5827fb3de497ac87b246/extensions/ViewController.swift#L18"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>